executor/cook/config.py

```diff
@@ -49,9 +52,22 @@ def initialize_config(environment):
     """Initializes the config using the environment.
     Populates the default values for missing environment variables.
     """
+    executor_id = environment.get('MESOS_EXECUTOR_ID', 'executor')
+    sandbox_directory = environment.get('MESOS_SANDBOX', '')
+    default_progress_output_name = '{}.progress'.format(executor_id)
```

Comment:
Nitpick: `f'{executor_id}.progress'`?

Reply:
Should we use f-strings throughout all the changes?

Reply:
We're using python 3.5 in the executor.